### PR TITLE
feat(workflow): Adds gh-action for building and publishing docker images to github container registry

### PR DESCRIPTION
**Pull request by deshmukhmayur**
# Resolves ONEPLAT-892

# Explain the feature/fix

Created github workflows for building and publishing docker images for one platform microservices.

## Does this PR introduce a breaking change

No.

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [x] Does the change have appropriate unit tests?
- [x] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demo'd and the design review approved?

### CODE REVIEW

**Comment by riginoommen** [comment]
Looks Good

**Review by riginoommen — APPROVED** [review]

File: packages/feedback-service/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-FROM node:alpine
+FROM node:12-alpine
```
Comment (riginoommen) [inline packages/feedback-service/Dockerfile:1]: Can we use 14 LTS version

Comment (deshmukhmayur) [reply packages/feedback-service/Dockerfile:1]: The microservices are built with 12, so we'll have to first make sure all the microservices work inn node:14.

After that, it's just about bumping the number from 12 to 14.

**Review by riginoommen — APPROVED** [review]

**Review by sayak-sarkar — APPROVED** [review]

**Comment by deshmukhmayur** [comment]
Converting the PR to draft.
Need some rethinkig for whether to replace the base image with [ubi8/nodejs-12](https://catalog.redhat.com/software/containers/ubi8/nodejs-12/5d3fff015a13461f5fb8635a?gti-tabs=unauthenticated) image.

**Pull request merged**